torch_xla/utils/dlpack.py

```diff
@@ -21,3 +23,28 @@ def from_dlpack(ext_tensor: Any):
     dlpack = ext_tensor
 
   return torch_xla._XLAC._from_dlpack(dlpack)
+
+
+def from_xla_cuda_to_cuda(tensor):
+  assert torch.cuda.is_available()
+  assert tensor.device.type == "xla", "The tensor is not an XLA tensor"
+  is_xla_cuda = True if xu.getenv_as("PJRT_DEVICE", str,
+                                     "").lower() == "cuda" else False
+  assert is_xla_cuda, "The XLA tensor is not on CUDA"
+  # consumer is torch, producer is torch_xla
+
+  # Similar logic as torch.utils.dlpack.from_dlpack
+  # https://github.com/pytorch/pytorch/blob/b0ef363972203b163cddc95e4c6054b8221c2300/torch/utils/dlpack.py#L114-L115
+  device_id = tensor.device.index
+  stream = torch_xla._XLAC._get_stream_for_cuda_device(device_id)
+  stream = 1 if stream == 0 else stream
```

Comment:
Could you explain what's going on, here? A comment would be nice.

Reply:
Sure. Per https://github.com/pytorch/pytorch/blob/b0ef363972203b163cddc95e4c6054b8221c2300/torch/utils/dlpack.py#L114-L115, "The array API specify that the default legacy stream must be passed with a value of 1 for CUDA". I added a comment.